### PR TITLE
Fix spotinst stack details

### PR DESCRIPTION
**Pull request by lmineiro**
This should fix the problem to use the auxiliary commands scale, patch and respawn
It stopped working with #535 and would fail if the Elastigroup component name was not `AppServer`

### CODE REVIEW

**Comment by lmineiro** [comment]
:+1:

File: senza/spotinst/components/elastigroup_api.py [hunk 1]
```diff
@@ -22,16 +24,22 @@ def __init__(self, account_id, access_token):
 
 
 def get_spotinst_account_data(region, stack_name):
-    '''
-    Extracts required parameters required to access SpotInst API
-    '''
+    """
+    Extracts the Spotinst API access token and cloud accoint ID required to use the SpotInst API
+    It returns those parameters from the first resource of Type Customer::elastigroup
```
Comment (pc-alves) [inline senza/spotinst/components/elastigroup_api.py]: Two typos:
 - 'cloud account ID'
 - 'resource of type Custom::elastigroup'

**Comment by pc-alves** [comment]
👍

**Comment by lmineiro** [comment]
:+1:

**Pull request merged**